src/reflect/scala/reflect/internal/Reporting.scala

```diff
@@ -78,6 +78,8 @@ import util.Position
  */
 abstract class Reporter {
   protected def info0(pos: Position, msg: String, severity: Severity, force: Boolean): Unit
+  def info1(pos: Position, msg: String, severity: Severity, force: Boolean): Unit =
```

Comment:
what is this?

Reply:
With current design i still cannot access info0.
But I can make this `protected[reporters]`.